client/extensions/woocommerce/style.scss

```diff
@@ -6,19 +6,13 @@
 }
 
 .woocommerce {
+	@import 'app/shipping/style';
```

Comment:
Agreed, I only brought it up because of the issues we were having styling the Zone modal :)